addons/escoria-core/game/core-scripts/esc_room.gd

```diff
@@ -113,22 +113,24 @@ func _ready():
 	else:
 		is_run_directly = true
 	
+	# Manage player location at room start
+#	if (escoria.globals_manager.get_global("ESC_LAST_SCENE") == null \
+#			or escoria.globals_manager.get_global("ESC_LAST_SCENE").empty()) \
```

Comment:
Remove this commented out code.

Reply:
Same. Sorry, I did some tests and forgot it :D